### DISCUSSION TITLE
Code formatting when using Claude 3.5 Sonnet

### DISCUSSION THREAD

**Discussion by Joakim-T** [open-webui/open-webui#5227] [General]
# Bug Report

## Installation Method

Docker

## Environment

Open WebUI v0.3.19
Claude added using LiteLLM

Windows 10
Chrome 128.0.6613.120

**Confirmation:**

- [X] I have read and followed all the instructions provided in the README.md.
- [X] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.
- [X] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

Formatted code output from LLM in a Run/Copy Code box

## Actual Behavior:

Unformatted code as normal chat text

## Description

After updating Open WebUI to the latest version code from Claude 3.5 is not formatted correctly.
Code from ChatGPT 4o looks fine.

**Bug Summary:**
[Provide a brief but clear summary of the bug]

## Reproduction Details

**Steps to Reproduce:**
See screenshots.
Ask Claude to write any code

## Logs and Screenshots

**Screenshots/Screen Recordings (if applicable):**

Claude 3.5

![claude35s](https://github.com/user-attachments/assets/817c01e5-81f1-4afb-94d7-5dfd422df8af)

ChatGPT 4o

![gpt4o](https://github.com/user-attachments/assets/bf6dd44c-d3cd-4794-808f-6a045ed25277)

**Comment by silentoplayz** [comment]
How do we know that claude-3.5 even provided the code wrapped in a codeblock based on your screenshot? We don't.

**Comment by Joakim-T** [comment]
Also tested using pipeline with the same problem so i assume it is not a LiteLLM problem.
https://github.com/open-webui/pipelines/blob/main/examples/pipelines/providers/anthropic_manifold_pipeline.py

Seems to only be happening with anthropic/claude-3.5-sonnet for some reason and i noticed it after the update from v0.3.16 (i think) to v0.3.19 this morning.

**I found a workaround however if anyone else is having issues.**
If i add "Correctly wrap any code to make it easier to read." to the initial prompt or in the system prompt  it seems to work as expected again.
Been using OpenWebUI for months and did not have a system prompt previously.

Very strange issue and Anthropic do not mention any changes in their API.
https://docs.anthropic.com/en/release-notes/api
